src/main/java/com/codecool/virtualpub/data/Customer.java

```diff
@@ -60,7 +60,7 @@ public void Drink(int alcoholUnit) {
         this.alcoholLevel += alcoholUnit;
     }
 
-    public String Speak() {
+    public String speak() {
         Random random = new Random();
```

Comment:
I can see that you created a field called `random` which is great, but then you shouldn't create this variable here. I recommend removing this line.